src/scaler/daemonscaler/mod.rs

```diff
@@ -75,7 +76,7 @@ impl<S: ReadStore + Send + Sync + Clone> Scaler for ActorDaemonScaler<S> {
         } else {
             spread_config
         };
-        self.config.spread_config = spread_config;
+        self.spread_config.spread_config = spread_config;
```

Comment:
Nit: I can hear the Go compiler yelling at me in my head that this stutters and I actually agree with it here. Should we maybe name a field differently somewhere?

Reply:
No I think renaming would be good, I'll address in a followup